modules/cypress/package.json

```diff
@@ -5,5 +5,8 @@
   "type": "module",
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
```

Comment:
Unfortunatelly, this has to be in dependencies, not devDependencies. devDependencies of your dependencies are not installed (applies in implementation projects like ZNZ or Kofola)